plugin/src/main/java/org/opensearch/ml/action/deploy/TransportDeployModelAction.java

```diff
@@ -23,6 +23,8 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import javax.swing.*;
```

Comment:
Can we avoid to import `javax.swing.*` here?

Reply:
Good catch! Removed this redundant line in the refresh!